spec/components/task_list/section_component_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'rails_helper'
+
+RSpec.describe TaskList::SectionComponent, type: :component do
+  let(:section) do
+    instance_double TaskList::Section, tasks: tasks, name: :about_your_client
+  end
+
+  let(:tasks) { [task] }
+
+  let(:task) do
+    instance_double(
+      Tasks::CapitalAssessment,
+      name: :capital_assessment,
+      status: status,
+      path: 'path/to/step'
+    )
+  end
+
+  let(:status) { TaskStatus::NOT_STARTED }
+
+  # why would we want the list (for the totals, but that is not this components job)
```

Comment:
What is this comment about?

Reply:
Thanks @EdwinKruglov . It should not be there. (It was a reminder to myself before taking a break I think :))